src/views/Play/Play.tsx

```diff
@@ -348,32 +355,31 @@ export class Play extends React.Component<{}, PlayState> {
         this.setState({show_other_boardsize_challenges: !this.state.show_other_boardsize_challenges});
     };
 
-    anyChallengesToShow = (challenge_list): boolean => {
-
-        return this.state.show_all_challenges && challenge_list.length || challenge_list.reduce( (prev, current) => {
+    anyChallengesToShow = (challenge_list: Challenge[]): boolean => {
+        return this.state.show_all_challenges && challenge_list.length as any || challenge_list.reduce( (prev, current) => {
             return prev || current.eligible || current.user_challenge;
         }, false );
     };
 
-    liveOwnChallengePending = (): any => {
+    liveOwnChallengePending = (): Challenge => { // a user should have only one of these at any time
         const locp = this.state.live_list.find((c) => (c.user_challenge));
         return locp;
     };
 
-    ownRengoChallengePending = (): any => {
-        const orcp = this.state.rengo_list.find((c) => (c.user_challenge));
+    ownRengoChallengesPending = (): Challenge[] => { // multiple correspondence are possible, plus one live
+        const orcp = this.state.rengo_list.filter((c) => (c.user_challenge));
+        //console.log("own rcp", orcp);
         return orcp;
     };
 
-    joinedRengoChallengePending = (): any => {
+    joinedRengoChallengesPending = (): any[] => { // multiple correspondence are possible, plus one live
```

Comment:
Oops I missed one - looks like `any[]` can be `Challenge[]`

Reply:
Heh heh, I'm the one that missed it.  I had to "chose ours" during the merge for this part of the code, I missed going back to that one.